src/main/resources/db/migration/V1__init.sql

```diff
@@ -35,12 +35,24 @@ CREATE TABLE contacts (
     ON DELETE CASCADE
 );
 
+CREATE TABLE approvements (
+  task_id      INT,
+  user_id      INT,
+  mark         ENUM ('GOOD', 'BAD', 'VIEWED') DEFAULT 'VIEWED',
```

Comment:
So...
We should add approvement as VIEWED and then update it to GOOD/BAD.
It is two methods and 4 queries to db